src/plugins/1/plugin-layout/prerequis/index.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: full.html.hbs
+title: Prerequisites
+description: how to create a custom plugin
+order: 0
```

Comment:
For the time being, the menu order is incorrect. I guess it should be 
1. Prerequisites
2. configuration
3. init

Further, as a nitpicking: can we rename this file to english?